ble_controller/manifest.yaml

```diff
@@ -0,0 +1,4 @@
+description: nRF Bluetooth LE Controller
+git_revision: cc500c1b497a4725efb3ce7a3be7d2d0539ea71c
+sub_version_number: '0xFFFF'
```

Comment:
Unrelated to the PR creation mechanism. I believe we should change this to "ll_subversion_number"